articles/quickstart/spa/vuejs/interactive.md

```diff
@@ -0,0 +1,180 @@
+  webOriginUrl: 'http://localhost:3000',
+  showWebOriginInfo: true
+}) %>
+
+## Install the Auth0 Vue SDK {{{ data-action=code data-code="index.js" }}}
+
+Auth0 provides a [Vue SDK](https://github.com/auth0/auth0-vue) to simplify the process of implementing Auth0 authentication and authorization in Vue 3 apps.
+
+Install the Auth0 Vue SDK by running the following commands in your terminal:
+
+```bash
+cd <your-project-directory>
+npm install @auth0/auth0-vue
+```
+
+### Register the plugin
+
+For the SDK to function properly, you must register the plugin with your Vue application using the following properties:
+
+- `domain`: The domain of your Auth0 tenant. Generally, you can find this in the Auth0 Dashboard under your Application's Settings in the Domain field. If you are using a [custom domain](https://auth0.com/docs/custom-domains), you should set this to the value of your custom domain instead.
+- `client_id`: The ID of the Auth0 Application you set up earlier in this quickstart. You can find this in the Auth0 Dashboard under your Application's Settings in the Client ID field.
+- `redirect_uri`: The URL in your application that you would like Auth0 to redirect users to after they have authenticated. This corresponds to the callback URL you set up earlier in this quickstart. You can also find this value in the Auth0 Dashboard under your Application's Settings in the Callback URLs field. Make sure what you enter in your code matches what you set up earlier or your users will see an error.
+
+The plugin will register the SDK using both `provide` and `app.config.globalProperties`, allowing the SDK to be used with both the [Composition API](https://vuejs.org/guide/introduction.html#composition-api) and [Options API](https://vuejs.org/guide/introduction.html#options-api).
+
+::::checkpoint
+
+:::checkpoint-default
+
+The plugin should now be properly configured. Run your application to verify that:
+- the SDK is initializing correctly
+- your application is not throwing any errors related to Auth0
+
+:::
+
+:::checkpoint-failure
+Sorry about that. Here's a couple things to double check:
+* make sure the correct application is selected
+* did you save after entering your URLs?
+* make sure the domain and client ID are set correctly
+
+Still having issues? Check out our [documentation](https://auth0.com/docs) or visit our [community page](https://community.auth0.com) to get more help.
+
+:::
+::::
+
+## Add login to your application {{{ data-action=code data-code="login.js" }}}
+
+Now that you have configured your Auth0 application and the Auth0 Vue SDK, you need to set up login for your project. To do this, you will use the SDK’s `loginWithRedirect` function that is exposed on the return value of `useAuth0`, which you can access in your component's setup function. It will redirect users to the Auth0 Universal Login page. and, after a user successfully authenticates, redirect then back to the callback URL you set up earlier in this quickstart.
```

Comment:
```suggestion
Next, you will set up login for your project. You will use the SDK’s `loginWithRedirect` function exposed on the return value of `useAuth0`, which you can access in your component's setup function. It will redirect users to the Auth0 Universal Login page. and, after a user authenticates, redirect then back to the callback URL you set up earlier in this quickstart.
```